tools/testrunner/testrunner.go

```diff
@@ -39,16 +39,31 @@ import (
 	"github.com/google/uuid"
 )
 
+const (
+	codeCoverageExtension = ".cover.out"
+	retriesFlag           = "--retries="
+	coverProfileFlag      = "-coverprofile="
```

Comment:
why this one has only one dash?

Reply:
`coverprofile` is from Go, `--retriesFlag` and `--junitfile` are out own invention. Not sure which one is "correct".